src/conversion_tests.h

```diff
@@ -0,0 +1,80 @@
+#ifndef CYCAMORE_SRC_CONVERSION_TESTS_H_
+#define CYCAMORE_SRC_CONVERSION_TESTS_H_
+
+#include <gtest/gtest.h>
+
+#include <boost/shared_ptr.hpp>
+
+#include "agent_tests.h"
+#include "context.h"
+#include "exchange_context.h"
+#include "facility_tests.h"
+#include "material.h"
+#include "test_context.h"
+#include "conversion.h"
+#include "pyne.h"
+
+namespace cycamore {
+
+class ConversionTest : public ::testing::Test {
+ public:
+  cyclus::TestContext tc;
+  TestFacility* trader;
+  cycamore::Conversion* conv_facility;
+  std::string incommod1, outcommod_name;
+  double throughput_val, input_capacity_val;
+  cyclus::Composition::Ptr recipe;
+
+  const double TEST_QUANTITY = 10.0;  
+
+  const int SIMULATION_DURATION = 3;
+  const double DEFAULT_POSITION = 0.0;
+  const double TEST_POSITION = 0.01;
+  
+  const double DEFAULT_THROUGHPUT = 10.0;
+  const double DEFAULT_INPUT_CAPACITY = 50.0;
+  
+  const std::string DEFAULT_CONFIG = 
+    "<incommods>"
+    "  <incommodity>incommod1</incommodity>"
+    "</incommods>"
+    "<outcommod>outcommod</outcommod>"
+    "<throughput>" + std::to_string(DEFAULT_THROUGHPUT) + "</throughput>"
+    "<input_capacity>" + std::to_string(DEFAULT_INPUT_CAPACITY) + "</input_capacity>";
+
+  const std::string INCOMMOD1 = "incommod1";
+  const std::string OUTCOMMOD_NAME = "outcommod";
```

Comment:
If you define these before `DEFAULT_CONFIG` can you use them in that definition to ensure that these commodities are the same ones mentioned there?